### PR TITLE
[Distributed test]:Enable ddp_control_flow tests for ROCm

### PR DESCRIPTION
**Pull request by jaglinux**
Signed-off-by: Jagadish Krishnamoorthy <jagdish.krishna@gmail.com>

### CODE REVIEW

**Comment by jaglinux** [comment]
Thanks to the fix @ https://github.com/pytorch/pytorch/pull/53160 , the 2 tests are passing when run in loop.

**Comment by jaglinux** [comment]
cc @jeffdaily @arindamroy-eng

**Review by rohan-varma — APPROVED** [review]
Thanks! Will wait for ROCM CI to pass before landing.

**Pull request closed without merging**
